webui/react/src/e2e/models/pages/WorkspaceDetails.ts

```diff
@@ -0,0 +1,44 @@
+import { BasePage } from 'e2e/models/common/base/BasePage';
+import { Pivot } from 'e2e/models/common/hew/Pivot';
+import { ModelRegistry } from 'e2e/models/components/ModelRegistry';
+import { TaskList } from 'e2e/models/components/TaskList';
+import { TemplateList } from 'e2e/models/pages/Templates/TemplateList';
+import { ResourcePoolsBound } from 'e2e/models/pages/WorkspaceDetails/ResourcePoolsBound';
+import { WorkspaceProjects } from 'e2e/models/pages/WorkspaceDetails/WorkspaceProjects';
+
+/**
+ * Represents the Workspaces page from src/pages/WorkspacesDetails.tsx
```

Comment:
previously, this had been implemented in `webui/react/src/e2e/models/components/WorkspacesList.ts` and `webui/react/src/e2e/models/components/WorkspaceDetails.ts`. I moved all the same components into this file so they can match the source code.